src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Shipping from '../Icons/Shipping';
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const Message = styled.div`
+  flex-grow: 1;
+  padding: 0 1rem;
+`;
+
+const ArrivalDates = styled.span`
+  font-weight: bold;
+`;
+
+const ShippingDetails = ({ minDate, maxDate, shippingPromo }) => (
+  <Wrapper>
+    <Shipping />
+    <Message>
+      <div>Ship to Me</div>
+      <span>Arrives </span>
+      <ArrivalDates>{`${minDate} - ${maxDate}`}</ArrivalDates>
+      <div>{shippingPromo}</div>
```

Comment:
Same as above. Besides that since it is optional prop checking if it exists is advised: `{shippingPromo && <Text>{shippingPromo}</Text>}`